### PR TITLE
MSEDO-229_Search_symptoms

### PR DESCRIPTION
**Pull request by JoaoLetra**

### CODE REVIEW

**Review by DanielFran — APPROVED** [review]

File: src/test/javascript/spec/app/entities/symptom/symptom.component.spec.ts [hunk 1]
```diff
@@ -29,11 +30,14 @@ describe('Component Tests', () => {
...
               queryParamMap: of(
-                jest.requireActual('@angular/router').convertToParamMap({
+                jest.requireActual('@angular/router').convertToParamMap(
+                  {},
+                  {
                   page: '1',
                   size: '1',
                   sort: 'id,desc',
-                })
+                }
```
Comment (DanielFran) [inline src/test/javascript/spec/app/entities/symptom/symptom.component.spec.ts:39]: Missing here the criterias

**Review by DanielFran — CHANGES_REQUESTED** [review]

File: src/main/webapp/app/entities/symptom/symptom.component.ts [hunk 2]
```diff
@@ -1,14 +1,14 @@
-import { Component, OnInit, OnDestroy } from '@angular/core';
-import { HttpHeaders, HttpResponse } from '@angular/common/http';
-import { Subscription } from 'rxjs';
-import { JhiEventManager, JhiParseLinks } from 'ng-jhipster';
-import { NgbModal } from '@ng-bootstrap/ng-bootstrap';
+import {Component, OnInit, OnDestroy} from '@angular/core';
```
Comment (sfdsilva) [inline src/main/webapp/app/entities/symptom/symptom.component.ts:1]: Are these changes lint friendly?

File: src/main/webapp/app/entities/symptom/symptom.component.ts [hunk 3]
```diff
@@ -1,14 +1,14 @@
...
 
-import { ISymptom } from 'app/shared/model/symptom.model';
+import {ISymptom} from 'app/shared/model/symptom.model';
 
-import { ITEMS_PER_PAGE } from 'app/core/config/pagination.constants';
-import { SymptomService } from './symptom.service';
-import { SymptomDeleteDialogComponent } from './symptom-delete-dialog.component';
+import {ITEMS_PER_PAGE} from 'app/core/config/pagination.constants';
+import {SymptomService} from './symptom.service';
+import {SymptomDeleteDialogComponent} from './symptom-delete-dialog.component';
```
Comment (sfdsilva) [inline src/main/webapp/app/entities/symptom/symptom.component.ts:11]: Are these changes lint friendly?

**Review by sfdsilva — APPROVED** [review]

File: src/test/javascript/spec/app/entities/symptom/symptom.component.spec.ts (hunk 1 above)
Comment (sfdsilva) [reply src/test/javascript/spec/app/entities/symptom/symptom.component.spec.ts:38]: ^ (I approved considering this fix)

**Comment by DanielFran** [comment]
@JoaoLetra just needd to fix the conflicts to be merged

**Pull request merged**